src/store/session/webstorage.ts

```diff
@@ -23,10 +23,16 @@ limitations under the License.
 
 import * as utils from "../../utils";
 import { logger } from '../../logger';
+import { MatrixEvent } from "../../models/event";
 
 const DEBUG = false;  // set true to enable console logging.
 const E2E_PREFIX = "session.e2e.";
 
+export type MatrixEventPOJO = keyof MatrixEvent & {
+    // eslint-disable-next-line camelcase
+    txn_id: string;
+};
```

Comment:
This looks to be unused? imported yet never used.

Also mixing a class instance with extra fields seems very wrong as that implies we mutate class objects.

Seems like you want IEvent?